test/unit_tests/test_metadata_table.py

```diff
@@ -0,0 +1,141 @@
+#! /usr/bin/env python
+#-----------------------------------------------------------------------
+# Description:  Contains unit tests for parse_metadata_file in the class
+#               MetadataTable in scripts file metadata_table.py
+#
+# Assumptions:  
+#
+# Command line arguments: none
+#
+# Usage: python test_metadata_table.py         # run the unit tests
+#-----------------------------------------------------------------------
+import sys
+import os
+import logging
+import unittest
+
+root_dir = os.path.dirname(os.path.abspath(__file__))
+scripts_dir = os.path.join(root_dir, "../../scripts")
```

Comment:
I was told to use `os.pardir` instead of `..`

Reply:
One could use `os.pardir` or `os.path.dirname` but I agree about not using `/` or `..` in python.
```
test_dir = os.path.dirname(os.path.abspath(__file__))
root_dir = os.path.dirname(os.path.dirname(test_dir))
scripts_dir = os.path.join(root_dir, "scripts")
```
or
```
test_dir = os.path.dirname(os.path.abspath(__file__))
scripts_dir = os.path.abspath(os.path.join(test_dir, os.pardir, os.pardir, "scripts"))
```